track_topic: /tracking
cmd_vel_topic: /cmd_vel
desired_xy_scale: 3.0
desired_center_x: 450
desired_center_y: 300
fy_pid:
  p: 0.0005
  i: 0.0
  d: 0.0001
  i_clamp_max: 0.0
  i_clamp_min: 0.0
fz_pid:
  p: 0.01
  i: 0.000
  d: 0.002
  i_clamp_max: 0.0
  i_clamp_min: 0.0
